cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -79,7 +79,10 @@ def _is_single(self) -> bool | None:
         """
         return (
             self.adata.uns["spatial"]["is_single"]
-            if hasattr(self.adata, "uns") and "spatial" in self.adata.uns and "is_single" in self.adata.uns["spatial"]
+            if hasattr(self.adata, "uns")
+            and "spatial" in self.adata.uns
```

Comment:
If I set `uns['spatial']` = pandas dataframe, I get this exception at this line:
`ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().`